Merge from DEV

tkc17 opened the pull request:
As we have an upcoming release, merge the pending changes in to REL.

tjobanpu committed:
disabled DFS channels (#6)

Temporary Fix till we address luci issues.
Disable DFS completely both in Ap and STA modes.

tjobanpu committed:
reverting the DFS commit as the STA mode DFS would not be working (#9)

Signed-off-by: Tushar Jobanputra <tushar.jobanputra@imgtec.com>

Commit:
Update README (#12)

abhijit-mahajani-imgtec reviewed (changes requested):
Just to keep rel branch with clean commit history, 
you can remove the DFS commit to add and then revert, effectively no change.
I meant just merge the Readme commit into REL

Commit:
uccp420wlan: Fixes and Feature additions

Release 6.9 Updates.

1. Advertise support for cloned  skbs to avoid memcpy for TCP data
   transmissions
2. Move the FW Loading to insmod from ifconfig wlan0 up (PR#11)
3. Enable RTS/CTS protection from Minstrel (needed a kernel bug fix).
4. Increase the QUEUE flush timeout to handle 2 VIF cases.
5. Debug - add separate section in mac_stats for RPU frame Drops
6. Re-organized the HAL header files in to hal.h, rpu.h, soc.h so that
   it is easier for SoC desginer/customer to understand their scope.
   Also moved all common things to hal.h from hal_hostport.h
7. Bumped version to 6.0.6

Change-Id: I0c5fc9c9e1494087766a0ee9ac77510b1427de5a

francois-berder committed:
firmware: Update to version 6.9

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

Commit:
Merge pull request #14 from francois-berder-imgtec/task-203

Update wifi driver to version 6.9

tkc17 commented:
Will close this and open a new PR with only README changes.
